src/core/linux/SDL_udev.c

```diff
@@ -309,7 +309,11 @@ SDL_UDEV_LoadLibrary(void)
 
     /* See if there is a udev library already loaded */
     if (SDL_UDEV_load_syms() == 0) {
+#ifdef SDL_VIDEO_DRIVER_ROCKCHIP
+	SDL_LogWarn(SDL_LOG_CATEGORY_INPUT, "ROCKCHIP platform, Fix me! (UDEV)");
```

Comment:
Are there any specific udev functions that are missing with Rockchip?

Reply:
No. I reported an issue some time ago, but the latest version seems to be resolved. I cannot reproduce it anymore. I thought it was wise to let the user know.  Remove it?

Reply:
Yes please.